crates/miden-lib/asm/miden/tx.masm

```diff
@@ -50,6 +50,32 @@ export.get_block_hash
     # => [BLOCK_HASH]
 end
 
+#! Returns the block timestamp of the reference block for this transaction.
```

Comment:
Should we mention that this is of type u32, and users can safely use u32 operations on it?